### PR TITLE
Turkish translation update

### PR DESCRIPTION
**Pull request by serkan-maker**

### CODE REVIEW

File: avidemux_tr.ts [hunk 1]
```diff
@@ -6322,60 +6963,123 @@ Yüklemek istiyor musunuz  ?
         <translation>TS Çoklayıcı</translation>
     </message>
 </context>
+<context>
+    <name>ffv1</name>
+    <message>
+        <location filename="../../../avidemux_plugins/ADM_videoEncoder/ffv1/ADM_ffv1Encoder.cpp" line="126"/>
+        <source>Golomb-Rice</source>
+        <translation>Golomb-Pirinç</translation>
```
Comment (eumagga0x2a) [inline avidemux_tr.ts]: Thank you, but Robert Rice, the author of this method, is not groats. Please fix.

**Comment by serkan-maker** [comment]
Thank you for the warning. I fixed it as Golomb-Rice
[avidemux_tr.ts.tar.gz](https://github.com/mean00/avidemux2_i18n/files/6968846/avidemux_tr.ts.tar.gz)

File: avidemux_tr.ts (hunk 1 above)
Comment (serkan-maker) [reply avidemux_tr.ts]: Thank you for the warning. I fixed it as Golomb-Rice

**Comment by eumagga0x2a** [comment]
Thank you for the update and for providing the file as resolving merge conflicts manually would have been not feasible:

```
grep -c "<<<<<<< HEAD" avidemux_tr.ts
545
```
I can only remind of [my request](https://github.com/mean00/avidemux2_i18n/pull/9#issuecomment-787499181). In doubt, please don't hesitate to request a refresh of translation files rather than doing it locally, but if you do have to regenerate translation files locally, any deviation in the lupdate command line from the upstream (`qt_update_pro.sh:75`) results in huge diffs at best.

**Pull request merged**